test/util/unit-helpers.js

```diff
@@ -77,5 +79,12 @@ function fillDate(formDOM, partialId, dateString) {
   });
 }
 
+export function mockFetch(returnVal) {
+  global.fetch = sinon.stub().returns(returnVal);
+}
+
+export function resetFetch() {
```

Comment:
I wrote these before I realized I didn't actually need them. Still, they might be helpful later. :man_shrugging: